src/BlazorWasmSdk/Tasks/BrotliCompress.cs

```diff
@@ -111,14 +110,15 @@ protected override string GenerateResponseFileCommands()
                 }
 
                 builder.AppendLine("-s");
-                builder.AppendLine(inputFullPath);
+                builder.AppendLine(Quote(inputFullPath));
 
                 builder.AppendLine("-o");
-                builder.AppendLine(outputFullPath);
+                builder.AppendLine(Quote(outputFullPath));
```

Comment:
That's correct - S.CL changed the logic (to something that's more reasonable cross-platform) and this just happened to trigger it. Longer-term resolutions for this might include:

* not doing this tool-based execution at all in this task, and instead doing the logic in _code_ in the task (unsure if this is intended or not?)
* MSBuild ToolTask helpers/support for this?

Reply:
We can't do the brotli compression on a task because it needs to support net462 and brotli is not available there :(

Reply:
Aha! I thought it might have been something like this. Thanks for clarifying.